cnd/src/http_api/orderbook.rs

```diff
@@ -156,6 +159,11 @@ pub async fn post_make_herc20_hbit_order(
     let reply = warp::reply::reply();
     let order: NewOrder = body.to_order();
 
+    order
+        .assert_valid_ledger_pair()
+        .map_err(problem::from_anyhow)
+        .map_err(warp::reject::custom)?;
```

Comment:
Should this check be in the order book instead of the controller?

Reply:
I'm inclined to leave the orderbook being permissive. Following the principle of keeping policy out of the library and in the application. (I don't actually know if this is a thing in application development, in the kernel they talk about this - policy stays in userspace)

Reply:
Interesting viewpoint. What does that mean for compatibility between nodes though?
If the policy is important for compatibility, it should go into the (not yet existent) spec which means it should also go into the library that implements the spec (the `comit` crate).

What do you think?